src/Entity/DigitalPost.php

```diff
@@ -100,6 +100,11 @@ class DigitalPost
      */
     private $totalFileSize = 0;
 
+    /**
+     * @ORM\OneToOne(targetEntity=CaseEvent::class, mappedBy="digitalPost", cascade={"persist", "remove"})
+     */
+    private $caseEvent;
+
```

Comment:
Do really need this property, i.e. will we ever need to find a case event given a digital post?

Reply:
Good point. Removed.

Reply:
I do want to link back to the case event from the digital post, so this one i think we should keep.

See https://github.com/itk-dev/naevnssekretariatet/blob/feature/TVIST1-743-case-events/templates/case/communication/digital_post/show.html.twig#L12-L16